### ISSUE TITLE
Issue with curl output in python image

### ISSUE THREAD

**Issue by mivanovvs** [docker-library/python#437]
I faced with curl issue in python image(python:latest). The problem related with thrash in output with -v option.
I just tried build other versions of curl manually and got the same situation. Tried with alpine:latest image and curl is working with normal output. Probably problem with some library which curl is using.

<details>
<summary> Example of output </summary>

```console
curl -v http://ip_rest:8080/logs 
* Expire in 0 ms for 6 (transfer 0x558cd58f2f50)
* Expire in 1 ms for 1 (transfer 0x558cd58f2f50)
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 1 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 1 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 2 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
* Expire in 0 ms for 1 (transfer 0x558cd58f2f50)
*   Trying 10.233.15.147...
* TCP_NODELAY set
* Expire in 200 ms for 4 (transfer 0x558cd58f2f50)
* Connected to IP-restrest (10.233.15.147) port 8080 (#0)
> GET /logs HTTP/1.1
> User-Agent: curl/7.64.0
> Accept: */*
> 
* HTTP 1.0, assume close after body
< HTTP/1.0 200 OK
< Date: Thu, 05 Dec 2019 12:28:50 GMT
< Server: WSGIServer/0.2 CPython/3.8.0
< content-type: application/json; charset=UTF-8
< content-length: 65
< 
{ [65 bytes data]
100    65  100    65    0     0   4642      0 --:--:-- --:--:-- --:--:--  4642
```

</details>

**Comment by wglambert** [comment]
This was fixed in `7.64.1` https://github.com/curl/curl/pull/3558
Debian Buster (what Python:latest uses) is at [7.64.0-4](https://packages.debian.org/buster/curl). So whenever Buster's `curl` is updated the change will be incorporated into the images

**Comment by wglambert** [comment]
Closing since this is an upstream-package issue, whenever that gets resolved the changes will propogate into the latest images shortly after
